packages/theme/src/cli/commands/theme/pull.ts

```diff
@@ -33,7 +33,7 @@ If no theme is specified, then you're prompted to select the theme to pull from
     }),
     nodelete: Flags.boolean({
       char: 'n',
-      description: 'Runs the pull command without deleting local files.',
+      description: 'Prevent deleting local files that have been removed remotely.',
```

Comment:
We don't know if the given files ever exist remotely, so I'd update this to something like this:
```suggestion
      description: 'Prevent deleting local files that that do not exist remotely.',
```